### DISCUSSION TITLE
Student developer pack issue

### DISCUSSION THREAD

**Discussion by rjahmad** [community/community#54848] [GitHub Education] [answered]
Hey i was creating my student developer pack but after upload my student id card proof its shows "your account is too recently created please try again in few days" i can try after 2 days but it still showing same error please anyone tell how can i get my student developer pack.

**Comment by iamreiyn** [comment, marked as answer]
Hi @rjahmad 

The exact amount of time it takes to become eligible for the GitHub Student Developer Pack after creating a new account can vary. In some cases, it may take only a few days, while in others it may take up to a week or more. It is recommended that you keep your GitHub account active and try again maybe in a week's time. Thank you.

**Reply by rjahmad** [reply]
Thanks fori forming me .

**Reply by iamreiyn** [reply]
> Thanks fori forming me .

@rjahmad Anytime, make sure to "Mark an Answer" to close out the discussion.

**Reply by askchandan** [reply]
My student dev pack has been approved and says the benefits will be available soon.  its already been 3 days , please help me recieve my benefits as soon as possible. Thankyou

**Reply by Thaiduderoy** [reply]
Hi there, I currently have my student benefits approved by Github but when I go to use Github copilot it says I still need to pay for it even though the student developer pack makes Github copilot free. Would there be any fix for this?

**Comment by sumihiri-fernando** [comment]
Hi ! I want to join the GitHub Student Developer Pack. But when I declare proof I m a student. This massege is desplaying. " Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously." Please help me!

**Reply by Nminsker** [reply]
I've encountered the same issue.

**Reply by DZetko** [reply]
Same here. Can @github, please, help us?

**Reply by aniketchauhan18** [reply]
same issue

**Reply by abmutungi** [reply]
same issue

**Reply by A-72** [reply]
Take another picture works for me.

**Reply by sakshinilawar20** [reply]
facing same issue

**Reply by AjinkyaUNCC** [reply]
same issue

**Comment by incendies** [comment]
same here dear @github could you help me too?

**Comment by Lavyatarah** [comment]
hello i wanted to get the domain name coupon and i cant find it :please help

**Comment by noorSEK** [comment]
![DSCN0038](https://github.com/community/community/assets/148883048/74cc146b-985c-42ca-a398-8c84a4c4c4ad)

**Comment by aryad14** [comment]
## Issue Regarding Application Submission for GitHub Student Developer Pack
I had submitted an application for GitHub Student Developer Pack 2 or 3 months earlier. Still I haven't received any mail regarding the status of my application. I check the **SPAM** folder too but still there are no mails.
Also I try to re-upload my identity proof as a student, I get the message as **Your uploaded proof has already been used previously.**

Here is a screenshot supporting the statement
![image](https://github.com/community/community/assets/95949958/275f9452-18f3-4136-b640-b6f26d339c89)
---
I would really appreciate if someone is able to help me solve this issue.

**Reply by A-72** [reply]
Try to take another picture.

**Comment by burning-painter** [comment]
Error my name only komal but forcing to add last name so not able to solve the issue 


Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

**Comment by FaxHack** [comment]
I have also not been able to get my benefits after waiting for almost a month now
